Technical Interview Coach Prompt

karkir0003 opened the pull request:
JIRA: Prompts-1234

**Purpose**
Jordan crowdsourced an awesome `tech-gpt-prompts` repo to share helpful prompts during his day to day as a software engineer.

I wanted to add a technical interview coach persona to the repository to give back to the High Growth Engineer community.

**Changes Made**
* Added a Technical Interview Coach directory with prompt/persona + usage instructions
* Minor update to main `README.md` with easy link to the new prompt

**Testing Plan**
* Created a gem in Gemini pro and tested out the persona. It works pretty good!

karkir0003 committed:
Create tech_interview_coach_prompt.md

karkir0003 committed:
Create setup_instructions.md

karkir0003 committed:
Update setup_instructions.md

karkir0003 committed:
Rename tech_interview_coach_prompt.md to tech-interview-coach-prompt.md

karkir0003 committed:
Update setup_instructions.md

karkir0003 committed:
Update and rename tech-interview-coach-prompt.md to tech-interview-coach-by-karthik.txt

karkir0003 committed:
Rename tech-interview-coach-by-karthik.txt to tech-interview-coach-by-karthik-subramanian.txt

karkir0003 commented:
@jordan-cutler feel free to take a look when you get time! Thanks for open sourcing this repo

karkir0003 committed:
Update setup_instructions.md

jordan-cutler commented on technical-interview-coach/setup_instructions.md:
```diff
@@ -0,0 +1,5 @@
+# How to setup Technical Interview Coach
```
can we name this README.md so it shows up when visiting the folder in Github?

karkir0003 replied:
renamed!

jordan-cutler commented on technical-interview-coach/setup_instructions.md:
```diff
@@ -0,0 +1,5 @@
+# How to setup Technical Interview Coach
+
+1. Pick your favorite foundational LLM (eg: Chat GPT, Claude, Gemini). I personally went with Gemini for my usage here since configuring gems is really easy here. If you are a student in university, Gemini offers 1 year of Pro tier for free
```
Let's add links to the appropriate places for ChatGPT, Claude, Gemini so it's clear exactly where you want them to go

karkir0003 replied:
added

jordan-cutler reviewed (commented):
Very nice! Thanks for doing this. Added a couple small suggestions

karkir0003 committed:
Rename setup_instructions.md to README.md

karkir0003 committed:
Update README.md

karkir0003 commented:
> Very nice! Thanks for doing this. Added a couple small suggestions

No problem! Thought it'd help the High Growth Engineer community. 

@jordan-cutler , I have gone ahead and addressed the other comments. Could you take a look when you get a chance

karkir0003 committed:
Update README.md

jordan-cutler reviewed (approved):
Awesome! Thanks so much for adding this one 🙂

karkir0003 commented:
> Awesome! Thanks so much for adding this one 🙂

no problem!
